From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Benjamin Marzinski <bmarzins@redhat.com>
Date: Mon, 23 Jun 2025 13:43:45 -0400
Subject: [PATCH] libmpathpersist: remove pointless update_map_pr ret value
 code

Don't set ret to MPATH_PR_SUCCESS, after the code just made sure that
it was already set to MPATH_PR_SUCCESS.

Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 libmpathpersist/mpath_persist.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/libmpathpersist/mpath_persist.c b/libmpathpersist/mpath_persist.c
index 1ce43de1..d9439292 100644
--- a/libmpathpersist/mpath_persist.c
+++ b/libmpathpersist/mpath_persist.c
@@ -906,8 +906,6 @@ int update_map_pr(struct multipath *mpp)
 		goto out;
 	}
 
-	ret = MPATH_PR_SUCCESS;
-
 	if (resp->prin_descriptor.prin_readkeys.additional_length == 0 )
 	{
 		condlog(was_set ? 1 : 3, "%s: No key found. Device may not be registered. ", mpp->alias);
